source/dynamic_data_attachment.md

```diff
@@ -0,0 +1,97 @@
+# Dynamic Data Attachment
+
+This support article illustrates how data from one project can be linked dynamically and referenced by another project(s). It also aims to demonstrate some practical examples that this feature supports.
+
+Now that this feature is available with KoBoToolbox, users can do everything that a [pulldata function](pull_data_kobotoolbox.md) does without having to maintain a separate CSV file. The data stored in the survey project (that the user intends to link) acts as a source file. Users could keep collecting data for their source files and simultaneously use them in another survey project(s). Thus, this feature should help users create and manage a longitudinal survey project _(longitudinal data is collected through a series of repeated observations of the same subjects over a short or long period)_. The best part of this feature is that it also works with a shared project where the project admin could also restrict specific permissions while sharing.
+
+Users can now retrieve any information (such as _text_, _integer_, _decimal_, _select_one response_, _select_multiple response_, _GPS points_) to a child project (aka round 2 project) that has already been collected and stored in a parent project (aka round 1 project). However, media information (such as _image_, _audio_, _video_, _uploaded files_, etc.) cannot be retrieved at the moment.
```

Comment:
Same comment as https://github.com/kobotoolbox/docs/pull/86#discussion_r667159833 regarding the use of "now". You can just say:
> Users can retrieve...

Reply:
Make sure to follow the convention now of using _child project_ and _parent project_ in italics.

Reply:
> ... _GPS points_) to a child project ...

to

> _GPS points_) in a _child project_ ...

Reply:
I would rather change:

> (aka round 2 project) ... (aka round 1 project)

To something like

> (e.g. phase 2) ... (e.g. phase 1)

Or something similar (unless there is a specific reason for having said that way)